src/transformState.js

```diff
@@ -4,8 +4,23 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  actions.forEach(action => {
+    if (action.type === 'addProperties') {
+      Object.assign(state, action.extraData);
+    }
+
+    if (action.type === 'removeProperties') {
+      action.keysToRemove.forEach(item => delete state[item]);
```

Comment:
If you iterate through keys, let's use `key` as the name of an element, and not abstract `item`